buildtools/pip/testdata/setup.py

```diff
@@ -0,0 +1,8 @@
+from setuptools import setup, find_packages
+
+setup(
```

Comment:
Another accepted and commonly used method to specify dependencies is like this:
```
    install_requires=['Cython>=0.25',
                      'h5py>=2.6',
                      'matplotlib>=3.0',
                      'numpy>=1.15',
                      'pandas>=0.19',
                      'scipy>=0.18',
                      'scikit-learn',
                      'scikit-image==0.14.2',
                      'seaborn>=0.7',
                      'spacy>=2.0',
                      'tqdm',
                      'tabulate>=0.7',
                      'tensorflow==1.12',
                      'PyYAML>=3.12'
                      ],
```
Currently the parser panics when trying to read the above data. Let me know if you want some examples.

Reply:
it actually doesn't panic -- we don't split by lines when loading the file, and just skip the `\n` characters while parsing. I'll add it to the test `setup.py` to make that expectation explicit, though

Reply:
Understood, I tested this and found a panic but it appears to be happening in a different place. Thanks for the explanation.